.travis.yml

```diff
@@ -1,25 +1,57 @@
-language: cpp
-cache: ccache
 os: linux
 dist: xenial
-matrix:
+jobs:
   include:
-    - name: "First commit message adheres to guidelines at <a href=\"https://goo.gl/p2fr5Q\">https://goo.gl/p2fr5Q</a>"
+    - stage: "Lint and Compile" 
+      name: "First commit message adheres to guidelines at <a href=\"https://goo.gl/p2fr5Q\">https://goo.gl/p2fr5Q</a>"
       if: type = pull_request
       language: node_js
       node_js: "node"
       script:
         - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
             bash -x tools/lint-pr-commit-message.sh ${TRAVIS_PULL_REQUEST};
           fi
+
     - name: "Linter"
       language: node_js
       node_js: "node"
-      env:
-        - NODE=$(which node)
+      install:
+        - make lint-py-build || true
+      script:
+        - NODE=$(which node) make lint-ci
```

Comment:
The commit message should mention that job now lints python scripts.

Reply:
Ack